features/features.json

```diff
@@ -1,4 +1,17 @@
  [
+  {
+    "title": "Upload Images with Scratch Assets in the forums by either clicking on the button or ctrl+y",
+    "description": "Use assets.scratch.mit.edu as an image host",
```

Comment:
The code all seems to work fine, but the title is too long, the description is too short, and too much information is given. Maybe a title such as "Direct image uploads in the forums" or something. For the description, keep in mind that these image uploads DO work for forum signatures.